db/migrate/20160808150745_add_metrics_status_to_ext_management_system.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetricsStatusToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :last_metrics_error, :text
```

Comment:
:+1:, though the `last_` part would keep it consistent with the `last_refresh` columns.

Btw, we used `last_refresh_error` to store the error value, and an empty `last_refresh_error` indicated that `last_refresh_date` was the date of the last *successful* refresh.

I'm fine with tracking metrics with three columns showing status, last updated, last success.

We can settle on the correct list of fields for a generic refactoring later, I think.